types/react/index.d.ts

```diff
@@ -3348,4 +3348,6 @@ declare global {
             view: React.SVGProps<SVGViewElement>;
         }
     }
+
+    export function unsable_act<T>(callback: () => T | PromiseLike<T>): PromiseLike<T>;
```

Comment:
This is not documented and its status unclear. Let's not add this now.

Reply:
How can we get the CI test to pass without it?